code/modules/mob/living/carbon/xenomorph/Evolution.dm

```diff
@@ -139,12 +139,15 @@
 	else
 		new_xeno.plasma_stored = new_xeno.plasma_max*(plasma_stored/plasma_max) //preserve the ratio of plasma
 
+	for(var/i in src.built_structures)
+		new_xeno.built_structures += built_structures[i]
```

Comment:
can't you just do `built_structures.Copy()`? also QDEL_NULL_LIST it once you're done